src/cascadia/Remoting/Peasant.idl

```diff
@@ -11,12 +11,15 @@ namespace Microsoft.Terminal.Remoting
 
         String[] Commandline { get; set; };
         String CurrentDirectory();
+
     };
 
     runtimeclass WindowActivatedArgs
     {
         WindowActivatedArgs(UInt64 peasantID, Guid desktopID, Windows.Foundation.DateTime activatedTime);
+        WindowActivatedArgs(UInt64 peasantID, UInt64 hwnd, Guid desktopID, Windows.Foundation.DateTime activatedTime);

```

Comment:
Interesting -- is this needed on the public surface? I only saw it used via `make_self`.

Reply:
ONCE AGAIN ignore me haha